### PR TITLE
fix: store task updates in correct schema property

### PR DESCRIPTION
**Pull request by Ayush5120**
Fixed background task progress by storing the output of the task at the correct root which is "task_outgoing".

### CODE REVIEW

File: pro_tes/tasks/track_task_progress.py [hunk 1]
```diff
@@ -2,36 +2,36 @@
...
-    self,  # pylint: disable=unused-argument
-    worker_id: str,
-    remote_host: str,
-    remote_base_path: str,
-    remote_task_id: str,
+        self,  # pylint: disable=unused-argument
+        worker_id: str,
+        remote_host: str,
+        remote_base_path: str,
+        remote_task_id: str,
```
Comment (uniqueg) [inline pro_tes/tasks/track_task_progress.py]: Did `black` auto-format that in this way? I'm pretty sure there should be only one level of indentation relative to the function `def` here...

**Review by uniqueg — CHANGES_REQUESTED** [review]
Looks good to me, thanks! Please double check the indentation. Specifically, please unindent the function args by one level and then run Black. Does it change again to two levels of indentation? If so, let me know and we'll just go with that. If it keeps the one level indentation, please commit and push and also let me know. Rest is fine, so I'll merge as soon as I hear from you, one way or the other.

File: pro_tes/tasks/track_task_progress.py (hunk 1 above)
Comment (Ayush5120) [reply pro_tes/tasks/track_task_progress.py]: Yes, you are right, I used black it reformatted to 1 level of indentation. fixing in the next commit

Comment (uniqueg) [reply pro_tes/tasks/track_task_progress.py]: Great, thanks

**Review by uniqueg — APPROVED** [review]

**Pull request merged**